apis/database/data.go

```diff
@@ -46,14 +46,23 @@ type DynamicGlobalProperties struct {
 }
 
 type Block struct {
-	Number                uint32               `json:"-"`
-	Timestamp             *types.Time          `json:"timestamp"`
-	Witness               string               `json:"witness"`
-	WitnessSignature      string               `json:"witness_signature"`
-	TransactionMerkleRoot string               `json:"transaction_merkle_root"`
-	Previous              string               `json:"previous"`
-	Extensions            [][]interface{}      `json:"extensions"`
-	Transactions          []*types.Transaction `json:"transactions"`
+	ID     int `json:"id"`
+	Result struct {
+		Extensions            []interface{} `json:"extensions"`
+		Previous              string        `json:"previous"`
+		Timestamp             string        `json:"timestamp"`
+		TransactionMerkleRoot string        `json:"transaction_merkle_root"`
+		Transactions          []struct {
+			Expiration     string          `json:"expiration"`
+			Extensions     []interface{}   `json:"extensions"`
+			Operations     [][]interface{} `json:"operations"`
```

Comment:
This does not finish off block.  I just thought that it finished off block.  I'm going to rescind this PR.  But this file here is exactly where I'm having trouble.  I want to cleanly import all of get_block into go variable types so that any information contained inside is readily accessible.